src/revChatGPT/V1.py

```diff
@@ -175,6 +175,9 @@ def ask(
                 continue
             if "data: " in line:
                 line = line[6:]
+                
+            if "Internal Server Error" in line:
```

Comment:
There are other ways an internal server error string could make it into the line such as by the Chatbot.